src/clitools/parser.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Yash Varshney
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#    http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+
+from src.model.spdx_no_assertion import SpdxNoAssertion
+from src.model.spdx_none import SpdxNone
+
+
+@click.command()
+@click.option("--file", prompt="File name", help="The file to be parsed")
+@click.option("--force", is_flag=True, help="print information even if there are some parsing errors")
+def main(file, force):
+    """
+    COMMAND-LINE TOOL for parsing file of RDF, XML, JSON, YAML and XML format.
+
+    To use : run `pyspdxtools_parser` using terminal or run `pyspdxtools_parser --file <file name>`
+
+    """
+    raise NotImplementedError("Currently, no parsers are implemented")
+
+    # Parse document and set as doc here
+    # First one to implement is the Json parser: https://github.com/spdx/tools-python/issues/305
+
+    print("doc comment: {0}".format(doc.comment))
+    print("Creators:")
+    for c in doc.creation_info.creators:
+        print("\t{0}".format(c))
+    print("Document review information:")
+    for review in doc.reviews:
```

Comment:
As we are deleting the old data model, I think we should also delete these old lines of code. They refer to fields that no longer exist. We could keep some kind of "rough skeleton" (e.g. print `creation_info`, `packages` and `files` or whatever) to keep the "main idea" of this script but I would delete most of the code.

Reply:
Good catch! I didn't look at these too closely (it's mostly just copied over), but we can remove it right away since it doesn't make sense in the new data model